src/qz/printer/action/WebApp.java

```diff
@@ -54,12 +55,14 @@ public void changed(ObservableValue<? extends Worker.State> ov, Worker.State old
             if (newState == Worker.State.SUCCEEDED) {
                 try {
                     Reflect.on(webView).call("setZoom", pageZoom);
+                    log.trace("Zooming in by x{} for increased quality", pageZoom);
                 }
                 catch(ReflectException e) {
                     log.warn("Unable zoom, using default quality");
                     pageZoom = 1; //only zoom affects webView scaling
                 }
 
+                log.trace("Setting HTML page width to {}", (pageWidth * pageZoom));
```

Comment:
A bit off topic, but how do we get these logs without firing a debug build?  These could be very useful when debugging with clients.